modules/veteran/app/models/veteran/service/representative.rb

```diff
@@ -1,12 +1,53 @@
 # frozen_string_literal: true
 
+require 'csv'
+
 module Veteran
   # Not technically a Service Object, this is a term used by the VA internally.
   module Service
     class Representative < ActiveRecord::Base
+      BASE_URL = 'https://www.va.gov/ogc/apps/accreditation/'
+
       self.primary_key = :representative_id
 
       validates_presence_of :poa
+
+      def self.reload!
+        array_of_hashes = fetch_data('orgsexcellist.asp')
+        array_of_hashes.each do |hash|
+          representative = Representative.find_or_initialize_by(representative_id: hash['Registration Num'])
+          representative.poa = hash['POA'].gsub!(/\W/, '')
+          representative.first_name = hash['Representative'].split(' ').second
+          representative.last_name = hash['Representative'].split(',').first
+          representative.phone = hash['Org Phone']
+          representative.save
+        end
+
+        Representative.where.not(representative_id: array_of_hashes.map { |h| h['Registration Num'] }).destroy_all
+
+        array_of_organizations = array_of_hashes.map do |h|
+          { poa: h['POA'], name: h['Organization Name'], phone: h['Org Phone'], state: h['Org State'] }
+        end.uniq.compact
+
+        array_of_organizations.each do |hash|
+          Organization.find_or_create_by(hash)
+        end
+      end
```

Comment:
The goal is to have this all run in a sidekiq job but haven't created that job because it's going to depend on what's decided from the implementation RFC. I'll look at bulk importing instead of iterating to save time.

Reply:
Made it a bulk import command.